src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2204,28 +2189,28 @@ private void ApplyLightLevelOption()
             case LightLevelOption.Day:
             {
                 dayButton.Pressed = true;
-                Editor.LightLevel = Editor.CurrentPatch.Biome.MaximumCompounds[sunlight].Ambient;
+                Editor.DayLightFraction = 1;
                 break;
             }
 
             case LightLevelOption.Night:
             {
                 nightButton.Pressed = true;
-                Editor.LightLevel = Editor.CurrentPatch.Biome.MinimumCompounds[sunlight].Ambient;
+                Editor.DayLightFraction = 0;
                 break;
             }
 
             case LightLevelOption.Average:
             {
                 averageLightButton.Pressed = true;
-                Editor.LightLevel = Editor.CurrentPatch.Biome.AverageCompounds[sunlight].Ambient;
+                Editor.DayLightFraction = Editor.CurrentGame.GameWorld.LightCycle.AverageSunlight;
```

Comment:
Is this always correct? Can the average sunlight fraction not vary by patch?

Reply:
This one is universal, patches just multiply `AverageSunlight` by their maximum sunlight to get their patch-specific average sunlight.

Reply:
It's universal *now* I was just wondering if it might not be the case in the future, though I guess it is kind of pointless to think about now if there's no ideas how this might potentially change in the future...